blocks/article-body-block/chains/article-body/_children/table.jsx

```diff
@@ -5,10 +5,10 @@ const Table = ({ element }) => {
   const { header, rows } = element;
 
   const tableHeaders = header.map((headerItem) => (
+    // eslint-disable-next-line jsx-a11y/control-has-associated-label
```

Comment:
This is to account for the removal of the `aria-label` below as setting an `aria-label` here actually hinders accessibility as by default assistive technologies would use the contents of the table header whereas with an `aria-label` all table headers would have the value specified in the `aria-label`